docs/apache-airflow/templates-ref.rst

```diff
@@ -33,25 +33,25 @@ Variables
 The Airflow engine passes a few variables by default that are accessible
 in all templates
 
-=========================================== ===================== ===================================================================
-Variable                                    Type                  Description
-=========================================== ===================== ===================================================================
```

Comment:
IMO [List Table](https://docutils.sourceforge.io/docs/ref/rst/directives.html#toc-entry-23) much easier to create / update rather then [Simple Tables](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#toc-entry-29)

Reply:
Agreed. I didn't know that existed! I have a real hard time with `.rst` in general, it's not very intuitive. Prefer `.md`, personally. 

That is a larger change than I'd like for this specific PR so I will not attempt to apply that currently, but will put it on my list of things for future PRs

Reply:
That's up to you. I'm initially found that entirely table changed, but seems like something indents broken and you have already fix it 👍